docs/natspec-format.rst

```diff
@@ -42,10 +42,6 @@ The following example shows a contract and a function using all available tags.
 
 .. note::
 
-  NatSpec currently does NOT apply to public state variables (see
```

Comment:
Has this been fixed already?

Reply:
I tested it and it was working, I think the person who fixed it just forgot to remove this